agent/xds/builtinextensions/ratelimit/copied.go

```diff
@@ -0,0 +1,59 @@
+package ratelimit
+
+import (
+	envoy_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoy_listener_v3 "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+	envoy_http_v3 "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
+	envoy_tls_v3 "github.com/envoyproxy/go-control-plane/envoy/extensions/transport_sockets/tls/v3"
+
+	"github.com/golang/protobuf/ptypes"
+
+	"github.com/golang/protobuf/proto"
```

Comment:
Could we use `google.golang.org/protobuf` instead? I believe @mkeeler is working on updating `github.com/goland/protobuf` references in our codebase.

Reply:
A PR has been merged to remove any usage of golang/protobuf and the linter will now complain about importing them.